pages/common/moro.md

```diff
@@ -1,23 +1,23 @@
-# Moro
+# moro
 
 > Track work time.
 
 - Run moro once right after starting the working day, and once more before leaving work:
 
 `moro`
 
-- To adjust the start of the working day:
+- Adjust the start of the working day:
 
 `moro hi {{09:30}}`
 
-- To adjust the end of the working day:
+- Adjust the end of the working day:
 
 `moro bye {{17:30}}`
 
-- To add a note on the working day:
+- Add a note on the working day:
 
 `moro note {{3 hours on project Foo}}`
 
-- To see the full report of all records and notes:
+- Show the full report of all records and notes:
```

Comment:
`Show the full report` -> `Show a full report`

Reply:
Agreed.